### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by paulrobertlloyd** [colinaut/action-table#8]
Hello again. This component is absolutely brilliant; basically every situation I come across, this component has me covered.

However! I spent a good number of hours trying to persist table filters across page views, only to find (while writing a very different issue) that this feature is already provided via the `store` attribute. Great! But also, gah! I think I was looking under documentation about filtering, as opposed to general guidance for the parent component.

As this project and the components mature, I wonder if there’s an opportunity to focus more on documentation? While the `README` is already quite detailed, maybe documentation could be broken down into the different sub-components, making it easier to navigate the different features?

Perhaps they could follow a structure and format similar to that for [HTML form components on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input):

- Attributes
- Methods
- Events
- Styling with CSS
- Accessibility
- Localisation
- Etc.

A number of examples and demos might may help too, ranging from simple to the more complex. I release this might be a me problem (not spotting something in the `README`), but thought I would share my frustration and this suggestion all the same.

Thanks again for your continued hard and excellent work on this project!

**Comment by colinaut** [comment]
Yeah the documentation definitely stand to be improved. I've got some other paid work on my plate at the moment — plus taxes weeeee! But I'd definitely like to make the documentation and examples better. Also, if you wanted to take a crack at improving them I'd happy take a PR.

P.S. Thanks for the compliment! I'm pretty happy with how this component has shaped up and it's already come in useful for one of my projects.
